.travis.yml

```diff
@@ -1,32 +1,29 @@
-sudo: false
-dist: trusty
 language: generic
+
 services:
   - docker
-cache: ccache
+
+cache:
+  directories:
+    - $HOME/.ccache
+
 env:
+  global:
+    - ROS_DISTRO="dashing"
```

Comment:
If you look at the github diff, you can see an example of how to do the matrix builds so we can easily test new distributions. Can you follow that styling? Eloquent is around the corner and there's a rosdistro file / docker containers so we should be able to do CI on Dashing and Eloquent.

Reply:
Everything looks good on my machine so we just need to make CI happy and we can be finished!

Reply:
I change the global ros distro to be in the matrix instead, couldn't find any good example on how it's done in other places, hopefully that is what you meant :)